### ISSUE TITLE
Error Contents: global name 'false' is not defined

### ISSUE THREAD

**Issue by schumi2004** [cflannagan/service.subtitles.addic7ed#1]
Manual search

17:15:51 T:1460   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.NameError'>
                                            Error Contents: global name 'false' is not defined
                                            Traceback (most recent call last):
                                              File "C:\Users\user\AppData\Roaming\XBMC\addons\service.subtitles.addic7ed\service.py", line 305, in <module>
                                                Search(item)
                                              File "C:\Users\user\AppData\Roaming\XBMC\addons\service.subtitles.addic7ed\service.py", line 206, in Search
                                                search_manual(item['mansearchstr'], item['3let_language'], filename)
                                              File "C:\Users\user\AppData\Roaming\XBMC\addons\service.subtitles.addic7ed\service.py", line 179, in search_manual
                                                return false
                                            NameError: global name 'false' is not defined
                                            -->End of Python script error report<--

**Comment by cflannagan** [comment]
Yep, I am also running into problems when trying to do a manual search. Will check it out

**Comment by cflannagan** [comment]
The code that I copied from the other repository, it was explicitly trying to 'return false' the first thing into implementation for manual search, so it appears to be intentionally disabled at present (I presume temporarily, for later yet-to-come implementation). I've replaced 'false' with 'False' (so it would stop throwing exception), and also added an user friendly message saying this is not enabled yet. I've committed this change into master, among other changes.

My intention is to bring manual search back eventually (no timeline yet). I have been preoccupied with DRYing the code up a bit (refactoring).
